tests/integration/run_importer.sh

```diff
@@ -6,6 +6,7 @@ TMPDIR=$(mktemp -d -t galaxy-importer-XXXXXXXX)
 python3 -m venv $TMPDIR/venv
 source $TMPDIR/venv/bin/activate
 pip install .
+cp -R skeleton $TMPDIR/skeleton
```

Comment:
we probably will not need integration test edits when this feature is defaulted to off